### PR TITLE
Added delete all replication rules capability

### PR DESCRIPTION
**Pull request by bexsoft**
## What does this do?
Added delete all replication rules capability

## How does it look?

Signed-off-by: Benjamin Perez <benjamin@bexsoft.net>

### CODE REVIEW

**Comment by harshavardhana** [comment]
We should perhaps ask for more confirmation @bexsoft - asking them to type "yes" or "delete"

**Comment by bexsoft** [comment]
> We should perhaps ask for more confirmation @bexsoft - asking them to type "yes" or "delete"

Fixed @harshavardhana 
Screen Shot 2022-02-15 at 14 14 56
Screen Shot 2022-02-15 at 14 14 48

File: swagger-console.yml [hunk 1]
```diff
@@ -934,6 +934,12 @@ paths:
           in: path
           required: true
           type: string
+        - name: delete_all
```
Comment (dvaldivia) [inline swagger-console.yml:937]: I think it's better to have a dedicated `delete_all` call

**Review by dvaldivia — CHANGES_REQUESTED** [review]

File: swagger-console.yml (hunk 1 above)
Comment (bexsoft) [reply swagger-console.yml]: Resolved

**Review by dvaldivia — APPROVED** [review]

**Review by adfost — APPROVED** [review]

**Pull request merged**